sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/_patch.py

```diff
@@ -3,69 +3,96 @@
+
+    def get_answers(self: "QuestionAnsweringClient", *args: Any, **kwargs: Any):  # type: ignore
+        """Deprecated wrapper for :meth:`QuestionAnsweringOperations.get_answers`.
 
-    def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, TokenCredential], **kwargs: Any) -> None:
-        try:
-            endpoint = endpoint.rstrip("/")
-        except AttributeError as exc:
-            raise ValueError("Parameter 'endpoint' must be a string.") from exc
-        super().__init__(
-            endpoint=endpoint,
-            credential=credential,  # type: ignore
-            authentication_policy=kwargs.pop("authentication_policy", _authentication_policy(credential, **kwargs)),
-            **kwargs
+        All positional / keyword arguments are forwarded to
+        ``self.question_answering.get_answers`` unchanged.
+        Additional keyword arguments (``**kwargs``) are forwarded unchanged to the
+        underlying operation.
+
+        :param args: Positional arguments forwarded unchanged to the underlying ``get_answers`` operation.
+        :type args: Any
+        :returns: Whatever the underlying operation returns (typically ``AnswersResult``).
+        :rtype: Any
+        :deprecated: Use ``client.question_answering.get_answers`` instead.
+        """
+        warnings.warn(
+            (
+                "QuestionAnsweringClient.get_answers is deprecated; use "
+                "client.question_answering.get_answers instead."
+            ),
+            DeprecationWarning,
+            stacklevel=2,
         )
-        self._default_language = kwargs.pop("default_language", None)
+        return self.question_answering.get_answers(*args, **kwargs)
 
+    def get_answers_from_text(self: "QuestionAnsweringClient", *args: Any, **kwargs: Any):  # type: ignore
+        """Deprecated wrapper for :meth:`QuestionAnsweringOperations.get_answers_from_text`.
 
-def patch_sdk():
-    pass
+        All positional / keyword arguments are forwarded to
+        ``self.question_answering.get_answers_from_text`` unchanged.
+        Additional keyword arguments (``**kwargs``) are forwarded unchanged to the
+        underlying operation.
+
+        :param args: Positional arguments forwarded unchanged to the underlying ``get_answers_from_text`` operation.
+        :type args: Any
+        :returns: Whatever the underlying operation returns (typically ``AnswersFromTextResult``).
+        :rtype: Any
+        :deprecated: Use ``client.question_answering.get_answers_from_text`` instead.
+        """
+        warnings.warn(
+            (
+                "QuestionAnsweringClient.get_answers_from_text is deprecated; use "
+                "client.question_answering.get_answers_from_text instead."
+            ),
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.question_answering.get_answers_from_text(*args, **kwargs)
 
+    if not hasattr(QuestionAnsweringClient, "get_answers"):
```

Comment:
Let's talk to the Azure board and see if we can drop these deprecated method from this beta version onwards